docker/test/integration/minifi/core/OPCUAServerContainer.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from .Container import Container
+
+
+class OPCUAServerContainer(Container):
+    def __init__(self, name, vols, network, image_store, command=None):
+        super().__init__(name, 'opcua-server', vols, network, image_store, command)
+
+    def get_startup_finished_log_entry(self):
+        return "TCP network layer listening on"
+
+    def deploy(self):
+        if not self.set_deployed():
+            return
+
+        logging.info('Creating and running OPC UA server docker container...')
+        self.client.containers.run(
+            "lordgamez/open62541",
```

Comment:
Is there a problem with the [official image](https://hub.docker.com/r/open62541/open62541)?

Reply:
There is a more detailed note about the issue in the PR description, but due to its size it would be problematic to use it in the CI. The change I proposed is already merged in the repository and will be available in the 1.3 release in the official dockerhub repository. It's currently in 1.3-RC1 phase at the moment so it should be available soon. We can replace it when we upgrade to that version.